.gitignore

```diff
@@ -1 +1 @@
-node_modules
+
```

Comment:
Remember, we don't ever want node_modules to be version controlled. Instead we want things in the dependencies lists and for every one to use `npm install` to install the dependencies for themselves. We put `node_modules` in the `.gitignore` file so that we never accidentally version control the folder.